packages/expo-dev-launcher/bundle/DevMenu.ts

```diff
@@ -0,0 +1,7 @@
+import { NativeModules } from 'react-native';
+
+export const DevMenu = NativeModules.ExpoDevMenu;
+
+export function isDevMenuAvailable(): boolean {
+  return !!DevMenu;
```

Comment:
I know I'm nitpicking 😄 
```suggestion
export function isDevMenuAvailable(): boolean {
  return !!NativeModules.ExpoDevMenu;
```